python/mirascope/llm/clients/openai/_utils.py

```diff
@@ -365,3 +365,39 @@ def create_format_tool_param(
             "strict": True,
         },
     )
+
+
+MODELS_WITHOUT_STRUCTURED_OUTPUT_SUPPORT = {
+    "gpt-4o-mini-audio-preview",
+    "o1-mini",
+    "gpt-3.5-turbo-1106",
+    "chatgpt-4o-latest",
+    "gpt-4o-2024-05-13",
+    "gpt-4-1106-preview",
+    "gpt-4o-mini-audio-preview-2024-12-17",
+    "o1-mini-2024-09-12",
+    "gpt-4o-audio-preview-2024-10-01",
+    "gpt-3.5-turbo",
+    "gpt-3.5-turbo-0125",
+    "gpt-4-turbo-2024-04-09",
+    "gpt-4-0125-preview",
+    "gpt-3.5-turbo-16k",
+    "gpt-4-turbo",
+    "gpt-4-0613",
+    "gpt-4-turbo-preview",
+    "gpt-4o-audio-preview-2024-12-17",
+    "gpt-4o-audio-preview-2025-06-03",
+    "gpt-4o-audio-preview",
+    "gpt-4",
+    "o1-preview-2024-09-12",
+    "gpt-4-0314",
+    "gpt-4-32k",
+    "codex-mini-latest",
+    "gpt-4-vision-preview",
+    "gpt-3.5-turbo-0301",
+    "o1-preview",
+    "gpt-3.5-turbo-16k-0613",
+    "gpt-4-32k-0314",
+    "gpt-3.5-turbo-0613",
+    "gpt-4-32k-0613",
+}
```

Comment:
Got this list by actual testing. Some of the models gave not found (maybe taken offline) but they were all models that, based on their siblings, also would not support structured outputs. So I included them here.